code/game/gamemodes/meteor/meteors.dm

```diff
@@ -319,31 +356,102 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 	if(!isspaceturf(T))
 		new /obj/effect/decal/cleanable/xenoblood(T)
 
-//Station buster Tunguska
-/obj/effect/meteor/tunguska
-	name = "tunguska meteor"
-	icon_state = "flaming"
-	desc = "Your life briefly passes before your eyes the moment you lay them on this monstrosity."
-	hits = 30
+//Blobby-Meaty Ore
+/obj/effect/meteor/meaty/mouldy
+	name = "mouldy ore"
+	desc = "Ewww."
+	color = "#654321"
+	meteorsound = 'sound/effects/splat.ogg'
+	meteordrop = list(/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/zombie, /obj/item/organ/tongue/zombie)
+	threat = 10
+	var/spawn_max = 4
+	var/blobbo_chance = 25
+
+/obj/effect/meteor/meaty/mouldy/meteor_effect()
+	..()
+	if(prob(blobbo_chance))
+		new /mob/living/simple_animal/hostile/blob/blobbernaut/independent(src.loc, src)
```

Comment:
can you let ghosts possess this guy?

Reply:
I thought of that but I wasn't sure if it was a good idea to do that.